openscenario/openscenario_interpreter/include/openscenario_interpreter/utility/execution_timer.hpp

```diff
@@ -103,11 +48,28 @@ class ExecutionTimer
 
     const auto end = Clock::now();
 
-    statistics_map[tag].add(end - begin);
+    statistics_map[tag](std::chrono::duration_cast<std::chrono::nanoseconds>(end - begin).count());
 
     return end - begin;
   }
 
+  void saveStatistics(boost::filesystem::path output_file)
```

Comment:
It seems redundant to have the class name in the member function name, so I think it would be better to just resolve it to `save`.

```suggestion
  auto save(const boost::filesystem::path & output_file) -> void
```

Reply:
fixed in [`ce1d416` (#1537)](https://github.com/tier4/scenario_simulator_v2/pull/1537/commits/ce1d416119ae0b78bde6fbc3f96e9a61d7795998)